src/test/java/seedu/address/model/transaction/TimestampTest.java

```diff
@@ -29,29 +29,29 @@ public void isValidTimestamp() {
         // invalid timestamps
         assertFalse(Timestamp.isValidTimestamp("")); // empty string
         assertFalse(Timestamp.isValidTimestamp(" ")); // spaces only
-        assertFalse(Timestamp.isValidTimestamp("2023-10-13")); // date only
-        assertFalse(Timestamp.isValidTimestamp("2023-32-32T12:34:56.789")); // invalid date
-        assertFalse(Timestamp.isValidTimestamp(" 2023-10-13T12:34:56")); // extra space
+        assertFalse(Timestamp.isValidTimestamp("2023-10-13T12:34")); // incorrect format
+        assertFalse(Timestamp.isValidTimestamp("12/13/2020 12:00")); // invalid date
```

Comment:
Consider apply boundary value analysis to add some more tests here, especially for all the invalid cases